### PR TITLE
Optimize Setup audit

### PR DESCRIPTION
**Pull request by megglos**
Went through all config pages to either remove the C7 label if not appropriate anymore or move it into sub-sections in case only some parts are C7 specific. Furthermore rearranged the Install/Config pages to also be aligned with the Navigation items used in other components. 
The special thing about Optimize is that I introduced sub-pages to Configuration to organise the relevant content.

### CODE REVIEW

File: docs/self-managed/optimize-deployment/optimize-explained/engine-data-deletion.md [hunk 1]
```diff
@@ -4,11 +4,9 @@ title: "Engine data deletion"
 description: "Explains how Optimize copes with the deletion of engine data."
 ---
 
-<span class="badge badge--platform">Camunda Platform 7 only</span>
-
 The engine slows down if the historic data grows significantly over time, particularly in cases where the amount of data streaming in each day is large. One solution to this is to remove old data from the engine on a regular basis, yet still importing the data to Optimize so it can be used for deeper analytics.
 
-To support the described use-case, Optimize does not care if you delete any data on the engine side. Specifically, Optimize does not sync with the engine on data deletion. If you want to remove any data from Optimize, you can either erase the data from Elasticsearch or use the [Optimize History Cleanup Feature](./../setup/history-cleanup.md).
+To support the described use-case, Optimize does not care if you delete any data on the engine side. Specifically, Optimize does not sync with the engine on data deletion. If you want to remove any data from Optimize, you can either erase the data from Elasticsearch or use the [Optimize History Cleanup Feature](./../../configuration/history-cleanup/).
```
Comment (megglos) [inline docs/self-managed/optimize-deployment/optimize-explained/engine-data-deletion.md:9]: linking to the `.md` file somehow didn't work what am I missing?

**Comment by megglos** [comment]
@akeller @christinaausley currently the link check fails as it expects some old links to be present based on the live sitemap, how do you deal with such cases?

**Comment by christinaausley** [comment]
Hi @megglos -- note that I've pushed a commit for my technical review to get this portion checked off. This of course will fail because I believe we need...drum roll...redirects!

For every link listed out on the build error script, we'll need to ensure it has an appropriate redirect. I'll work to implement this today. If I still can't get a clean build, we'll loop in @akeller for additional review.

**Review by christinaausley — APPROVED** [review]
This looks good on my end. Made a few technical adjustments in my commit, including:
- Sentence case structure
- Adding periods for complete sentences
- Uppercase Secret rather than lowercase secret
- Event-based processes rather than event based processes

**Comment by megglos** [comment]
@christinaausley perfect, thank you so much for the immediate help and review!

**Pull request merged**